docs/tutorial/expressjs.rst

```diff
@@ -0,0 +1,430 @@
+.. _build-a-rock-for-an-expressjs-application:
+
+Build a rock for an ExpressJS application
+-----------------------------------------
+
+In this tutorial, we'll create a simple ExpressJS application and learn how to
+containerise it in a rock with Rockcraft's
+:ref:`expressjs-framework <expressjs-framework-reference>` extension.
+
+Setup
+=====
+
+.. include:: /reuse/tutorial/setup.rst
```

Comment:
For the Docker installation, after the newgroup command a new terminal is spawned. So if someone copies the entire block and executes it at once, the following commands won't run

Reply:
That seems like it should be fixed as a separate PR, would you like to create an issue for it against this repository please?